mabot.py

```diff
@@ -0,0 +1,664 @@
+        if len(self.waitfor) == 0:
+            options = [set(v) for v in self.voted.values()]
+            u = set.intersection(*options)
+            if len(u) > 0:
+                self.result = [self.options[k] for k in list(u)]
+                self.status = self.STATUS[3]
+            else:
+                self.status = self.STATUS[4]
+
+    def vote_done(self, username, options):
+        # print('here voted:', options, len(self.options))
+        if max(options) == len(self.options):
+            self.voted[username] = []
+        else:
+            self.voted[username] = [int(o) for o in options]
+        self.waitfor = [i for i in self.waitfor if i != username]
+
+        self.check_results()
+        self.update()
+
+    def get_info(self):
+        empty_margin = '                           '
+        empty_line = f'\n{empty_margin}[empty]'
+        respond = f'''
+        <b>Meeting #{self.index}</b> <b>"{self.meeting_name}"</b> from <b>{self.user_firstname} ({self.username})</b> 
+        
+               created: {self.creating_time:%d.%m.%y %H:%M}
+               espired: {self.actuality_up_to:%d.%m.%y %H:%M}
+              duration: {self.duration} hours         {"[/change_duration]" if self.status == self.STATUS[0] else ""}
+    current status: <b>{self.status}</b>       [{"/run" if self.status == self.STATUS[0] else ""} /archivate]
+               invited:'''
+        if len(self.invited) != 0:
+            respond += ''.join([f'\n{empty_margin}-<b>{i}</b>' for i in self.invited])
+        else:
+            respond += empty_line
+        if self.status == self.STATUS[0]:
+            respond += f'\n{empty_margin}  [/add_persons /remove_persons]'
+
+        respond += \
+        f'''
+               options:'''
+        if len(self.options.items()) != 0:
+            respond += ''.join([f'\n{empty_margin}<pre>[{k}]</pre> ' + v.strftime("%d.%m.%Y %H:%M") for k, v in self.options.items()])
+        else:
+            respond += empty_line
+        if self.status == self.STATUS[0]:
+            respond += f'\n{empty_margin}[/add_options /remove_options]'
+
+        respond += \
+        f'''
+        already voted:'''
+        if len(self.voted.items()) != 0:
+            respond += ''.join([f'\n{empty_margin}- {k} for <pre>{v if len(v)>0 else "None above"}</pre>' for k, v in self.voted.items()])
+        else:
+            respond += empty_line
+
+        respond += \
+        f'''
+          waiting for:'''
+        if len(self.waitfor) != 0:
+            respond += ''.join([f'\n{empty_margin}- {i}' for i in self.waitfor])
+        else:
+            respond += empty_line
+
+        if self.status in {self.STATUS[3], self.STATUS[4]}:
+            respond += \
+            f'''
+            
+YOU'VE GOT RESULT: <b>{self.status}</b> '''
+        if self.status == self.STATUS[3]:
+            respond += 'Confirmed options: ' + ', '.join([f'{t.strftime("%d.%m.%Y %H:%M")}' for t in self.result])
+
+
+        return respond
+
```

Comment:
According to PEP8, there should be 2 blank lines before and after class/function definition